pkg/agent/endpoints/workload/handler.go

```diff
@@ -145,22 +145,27 @@ func (h *Handler) FetchJWTBundles(req *workload.JWTBundlesRequest, stream worklo
 
 // ValidateJWTSVID processes request for JWT SVID validation
 func (h *Handler) ValidateJWTSVID(ctx context.Context, req *workload.ValidateJWTSVIDRequest) (*workload.ValidateJWTSVIDResponse, error) {
+	log := h.Log.WithField(telemetry.Method, telemetry.ValidateJWTSVID)
 	if req.Audience == "" {
+		log.Error("audience must be specified")
 		return nil, status.Error(codes.InvalidArgument, "audience must be specified")
 	}
+
+	log = log.WithField(telemetry.Audience, req.Audience)
 	if req.Svid == "" {
+		log.Error("svid must be specified")
 		return nil, status.Error(codes.InvalidArgument, "svid must be specified")
 	}
 
 	_, selectors, metrics, done, err := h.startCall(ctx)
 	if err != nil {
+		log.WithError(err).Error("Failed to validate JWT SVID during context parsing")
```

Comment:
we should be consistent in regards to `JWT SVID` v.s. `JWTSVID`  (and `JWT-SVID` for that matter)

Reply:
Good point. I see that the terminology is currently inconsistent across the code comments, log messages, and documentation as is on master. Just confirming, are you asking me to change only the instances I'm touching in this PR, or in all places?

It looks like we should probably stick to `JWT-SVID` when referring to the identity document construct and `JWTSVID` when referring to the specific types defined in the node API proto and SPIRE codebase.

Reply:
The workload handler at a minimum should probably be consistent. I haven't looked though to know if it was already consistent or not. Your calculus for deciding which term to use sounds good.

Reply:
Updated instances in the code I touched and other places in the handler files I touched to be consistent in using `JWT-SVID` over `JWT SVID` to align with the terminology defined in the SPIFFE specification. If we want to clean up other usages of `JWT SVID`, we can probably create a new issue for that work since this PR is already pretty broad in scope.